### ISSUE TITLE
Maintainers needed

### ISSUE THREAD

**Issue by sbertix** [TheM4hd1/SwiftyInsta#204]
[_tl;dr available at the bottom_]

Hey 😊 

As most of you know by now, I'm no longer actively maintaining this repo.\
After soloing this for basically a year (_officially_ since `2.0.0`), I realized the "Instagram Private API"-scene in Swift needed some fresh air.

**SwiftyInsta** started as a porting of the (in)famous **mpg25** library, back in March, last year.\
@TheM4hd1 did a fantastic job providing for a solid `1.0.0` version, and as I had worked on a similar (private) project myself, I've decided, as the summer came and went, to join him in his efforts, and write `2.0.0`.

Since then a lot I've been working hard to improve it even further.\
**SwiftyInsta** gained multi-account supports (it used to rely entirely on singletons, meaning one account at a time), a web view based authentication, an improved `Endpoint` scripting and request manager, better paging, etc.

Unfortunately, as time went by, Swift evolved a lot, and regardless of all my efforts, **SwiftyInsta** could not, and cannot, keep up with it.

So I've decided to start from scratch and come up with an alternative solution, not plagued by that same "rigidity" in code.\
I've split the `URLSession`-client logic into [**ComposableRequest**](https://github.com/sbertix/ComposableRequest), which can be used to develop any sort of API client in Swift, and Instagram implementation in [**Swiftagram**](https://github.com/sbertix/Swiftagram): it's modular, it's efficient and it's written in modern Swift, with a modern dependency manager support (no more `CocoaPods` hell):
- cancel request, pause them, resume them…
- out-of-the-box **Combine** publishers
- improved web view based authentication
- **working** non-web view based authentication with username and password for custom experiences or `watchOS` and `tvOS` support
- modular and customizable `Authenticator`s, "write your own implementation if you need to".
- more endpoints
- modular and customizable `Storage` for `Secret`s (i.e. the authentication cookies): `UserDefaults`, keychain, or write your own one
- etc.

This means that, as of last month, I'm only actively fixing critical bugs (meaning they affect everyone and they preclude the usage of the entire library, so an endpoint not working is not one, for instance).
**SwiftyInsta** still has a lot of issues to fix:
- the non-web view based implementation does not work
- web view based implementation does not work as intended and does not allow for encrypted method to be called (e.g. following and unfollowing profiles is broken)
- some endpoints need updating

PRs and maintainers to fix this all are more than welcome.\

Thanks again to @TheM4hd1 for welcoming my support one year ago, and for the work he had done.\
And I genuinely hope my contributions helped you all 💪 

---

**tl;dr**: as I'm working on a more modern solution, called [**Swiftagram**](https://github.com/sbertix/Swiftagram), **SwiftyInsta** now needs maintainers. Check out some of the things that need working on above.
